ocs_ci/utility/users.py

```diff
@@ -0,0 +1,148 @@
+import os
+import random
+import string
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import exec_cmd
+
+
+def add_htpasswd_user(username, password, htpasswd_path):
+    """
+    Create a new user credentials with provided username and password.
+    These will be saved in file located on htpasswd_path. The file will
+    be created if it doesn't exist.
+
+    Args:
+        username (str): Name of a new user
+        password (str): Password for a new user
+        htpasswd_path (str): Path to httpasswd file
+
+    """
+    if os.path.isfile(htpasswd_path):
+        cmd = ['htpasswd', '-B', '-b', htpasswd_path, username, password]
+    else:
+        cmd = ['htpasswd', '-c', '-B', '-b', htpasswd_path, username, password]
+    exec_cmd(cmd)
+
+
+def create_htpasswd_secret(htpasswd_path, replace=False):
+    """
+    Create or update htpass-secret secret from file located on htpasswd_path.
+
+    Args:
+        htpasswd_path (str): Path to httpasswd file
+        replace (bool): If secret already exists then this will replace it
+
+    """
+    kubeconfig = os.getenv('KUBECONFIG')
+    if replace:
+        replace = (
+            f" --dry-run -o yaml | oc replace --kubeconfig {kubeconfig} -f -"
+        )
+    else:
+        replace = ''
```

Comment:
I would rather use different variable than the argument one, something like
```suggestion
    replace_str = ''
    if replace:
        replace_str = (
            f" --dry-run -o yaml | oc replace --kubeconfig {kubeconfig} -f -"
        )
```

And of course than later on use replace_str